### ISSUE TITLE
BatchNorm2d running_var uses the biased variance; match or document the unbiased convention

### ISSUE THREAD

**Issue by vyncint** [vyncint/oxmera#64]
## Summary

PyTorch normalises with the biased batch variance but updates `running_var` with the unbiased estimate (`n / (n - 1)`). Here the same biased `var` feeds both, so eval-mode outputs differ from a checkpoint trained elsewhere by `sqrt((n - 1) / n)` per channel, and nothing documents the choice.

## Where (main at 853a224)

```rust
// crates/oxmera-nn/src/norm.rs:113-129
let var = centered.mul(&centered)?.mean_keepdim(&[0, 2, 3], true)?;
...
*rv = rv.mul_scalar(1.0 - self.momentum)?.add(&flat_var.mul_scalar(self.momentum)?)?;
```

`docs/LIMITATIONS.md` documents that running statistics are not serialised, not this.

## Fix

Scale the running-variance update by `n / (n - 1)` where `n = N·H·W` (guarding `n == 1`), or state the convention next to the safetensors note in LIMITATIONS and in the `BatchNorm2d` docs. Matching is preferable: it is what every imported checkpoint assumes.

## Done when

A test trains one step on a known batch and asserts `running_var` against the hand-computed unbiased value.
